extensions/control-plane/api/management-api/contract-definition-api/src/main/java/org/eclipse/edc/connector/api/management/contractdefinition/model/ContractDefinitionRequestDto.java

```diff
@@ -46,6 +47,7 @@ public class ContractDefinitionRequestDto {
     @NotNull(message = "criteria cannot be null")
     private List<CriterionDto> criteria = new ArrayList<>();
     @Positive(message = "validity must be positive")
+    @Max(value = 315360000L, message = "validity should not be longer than 10 years (315.360.000 seconds)")
```

Comment:
Why is a valid contract restricted to 10 years?

Reply:
I thought it would be good to have a restriction related to the contract definitions times, in order to avoid mistakes leading to long times (e.g 99 years of a contract definition) but I can also either increase the number or simply remove this constraint.

Reply:
hey @jimmarino , it's just an arbitrary big value just to avoid the overflow at the calculation time when building up the catalog, we could set it way bigger, the important thing is that `duration + currentTimestamp < Long.MAX`. What could be a reasonable value?

Reply:
Long.MAX and then catch this case. We shouldn't impose a time restriction other than what the computing platform can handle. Long.MAX is good to represent effectively infinite time since the sun will burn out before we reach that time :-)

Reply:
Ok, so your proposal is to remove the validation, because `Long.MAX_VALUE` is already implicitly validated (if it's bigger it cannot deserialize the json correctly). I'm ok with removing this annotation @diegogomez-zf